_posts/en/2020-05-20-Team-Topologies-Whitelisting-app.md

```diff
@@ -3,59 +3,61 @@ layout: post
 title: "Team Topologies Inspired Whitelist App Development"
 ref: tt-whitelist
 lang: en
-author: "IT Strategy Team"
+author: "Jayson McIntosh"
 date: "2020-05-20"
 last_modified: "2020-05-20"
 excerpt_separator: <!--more-->
 ---
 
+A summary of the ideas and concepts used by the IT Strategy team and DTS to deliver the Whitelist application using DevOps inspired principles and ideas. By using this model, the team was able to demo an application to the CIO within 10 days of starting work, and had users on the cloud testing the application within 2 hours of PB being available.
+
+<!--more-->
+
 ## Context
 
 During the COVID19 situation there was an urgent need for an application to assist with the management of important information regarding the internal use of resources within the department. An application was built to avoid compiling an unmanageable list of spreadsheets with inconsistent data. The IT Strategy Team alongside DTS worked to create a solution using modern approaches and methodologies, which are described below. By using the below model, the team was informally formed on a Thursday late afternoon and was able to begin demoing the application to users the following Monday -- four days later (having worked through the weekend). By Tuesday, five days after the work began, the team had several repositories worth of work. An automated deployment pipeline was created in order to deploy changes to a development environment. Staging and production environments were also set up though at the time deployment to these was still manually triggered. On the tenth day of work the team was able to demo the application to the CIO. Less than a month later, the PB Cloud environment became available at 11am, and within minutes we had the environment up and running, and were able to have users on the application to test by 1:00pm, only two hours later.
```

Comment:
I think it is COVID-19?  With a hyphen?

Reply:
First sentence could be more clear....perhaps.... "...there was an urgent need for an application to help manage information about IT personnel within the department."  (resources is vague - what type of resources?)

Reply:
In the second sentence, maybe we should upfront say the application was build using open source software?

Reply:
also re: the above comment, maybe we could also add a nod to the TB policies.. e.g.Mandatory Procedures for Enterprise Architecture Assessment which require the use of Open Standards and Solutions by Default

Reply:
Added COVID -> COVID-19

Reply:
> In the second sentence, maybe we should upfront say the application was build using open source software?

Added

Reply:
> First sentence could be more clear....perhaps.... "...there was an urgent need for an application to help manage information about IT personnel within the department." (resources is vague - what type of resources?)

Not just IT, it's for the entire org. Added human resources